package.json

```diff
@@ -116,6 +116,7 @@
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
     "ember-responsive": "^3.0.6",
+    "ember-router-service-refresh-polyfill": "^1.0.1",
```

Comment:
sneaky :D, was expecting this in a seperate PR.
Guess we'll just go for it now and plan refactor stories later.

Reply:
Had to choose between adding it in now, or reviving `cases/case/subcases/subcase/route.js` just to have some kind of refresh action (and then also make sure it doesn't have name collissions, make sure it's being called correctly from the route, ...) so I thought just adding it was the easiest solution :P